lib/api/core/hotelClerk.js

```diff
@@ -497,8 +527,18 @@ class HotelClerk {
       throw new NotFoundError(`Unsubscribe error: not subscribed to ${roomId}`);
     }
 
-    if (notify) {
-      this.kuzzle.pluginsManager.trigger('core:hotelClerk:removeRoomForCustomer', {requestContext, roomId});
+    return this._removeRoomForCustomerWithoutThrowing(requestContext, roomId, notify);
+  }
+
+  _removeRoomForCustomerWithoutThrowing (requestContext, roomId, notify = true) {
```

Comment:
what's the goal of this method ?
As far as I read, it seems to only called by `_removeRoomForCustomer`, which already throw exceptions for the cases